### ISSUE TITLE
Support arbitrary versions for deploying to /trunk

### ISSUE THREAD

**Issue by ntwb** [stephenharris/grunt-wp-deploy#32]
I've just taken this repo for a test spin with BuddyPress:

BuddyPress utilises `/trunk` to test alpha, beta, and trunk versions.
• https://plugins.trac.wordpress.org/browser/buddypress/trunk/bp-loader.php#L18

Currently BP trunk is version `2.8.0-alpha`
https://buddypress.trac.wordpress.org/browser/trunk/src/bp-loader.php#L18

BuddyPress' most recent release is `2.7.4` 
• https://wordpress.org/plugins/buddypress/

With the following configuration of grunt-wp-deploy:

```js
wp_deploy:{
	trunk: {
		options: {
			build_dir: 'build',
			plugin_main_file: 'bp-loader.php',
			plugin_slug: 'buddypress',
			tmp_dir: 'tmp-wp-deploy'
		}
	}
}
```

I get a version mismatch:
```shell
Running "wp_deploy:trunk" (wp_deploy) task
[?] What's your SVN username? fake-netweb-username
>> Readme.txt version: 2.7.4
>> buddypress.php version: 2.8.0-alpha
Warning: Versions do not match: Use --force to continue.
```

It would be great to be able to support the above, I've a suspicion that from my initial poking around this won't be straightforward as the logic between _tags_ and _trunk_ will need further separation, I'll continue to poke around and see if anything pops up.

**Comment by stephenharris** [comment]
There's a `deploy_tag` setting - when set to false it will not deploy a new tag and shouldn't check the plugin versions.

It's intended when you want to update the plugin 'tested up to' version without actually deploying a new release.

That seems like it might be what you need?

**Comment by ntwb** [comment]
It continues to to run the version check and fails:

I updated my config to:
```js
wp_deploy:{
	trunk: {
		options: {
			build_dir: 'build',
			deploy_tag: false,
			deploy_trunk: true,
			plugin_slug: 'buddypress',
			tmp_dir: 'tmp-wp-deploy'
		}
	}
}
```

Results:

```shell
$ grunt wp_deploy 
Running "wp_deploy:trunk" (wp_deploy) task
[?] What's your SVN username? some-fake-username
>> Readme.txt version: 2.7.4
>> build/buddypress.php version: 2.8.0-alpha
Warning: Versions do not match: Use --force to continue.

Aborted due to warnings.
```

**Comment by stephenharris** [comment]
What version are you using? Here's where the version check is run but it should be skipped if the `deploy_tag` option is set to false: https://github.com/stephenharris/grunt-wp-deploy/blob/master/tasks/wp_deploy.js#L107

I've tested this manually the option was initially added, but I'll had an automated test to demonstrate this as well.

**Comment by stephenharris** [comment]
My mistake, I had some commits on the master branch that hadn't been published. You were mostly likely using 1.2.0, which did not have the fix for this issue. I have just released 1.2.1, which should skip the check if `deploy_tag` is set to `false`.

**Comment by ntwb** [comment]
Awesome, thanks @stephenharris version check now bypassed 👍
